lua/cscope/init.lua

```diff
@@ -367,10 +367,29 @@ local cscope_project_root = function()
 		if path == "/" then
 			return nil
 		end
-		path = path:match("^(.*)/")
+		if vim.fn.has("win32") and path:len() <= 2 then
```

Comment:
I was thinking something like this. It'll keep win and nix code separate -

```lua
if vim.fn.has("win32") then
  if path:len() <= 2 then
    return nil
  end
  path = path:match("^(.*)[/\\]")
else
  if path == "/" then
    return nil
  end
  path = path:match("^(.*)/")
end
```

Does https://github.com/dhananjaylatkar/cscope_maps.nvim/pull/33/files#diff-5cc864f78e58f85e7553ce50f9b7d9fca73d090527c72df47b59130683da318cR404 work on windows? I used "/" to concat the path

Reply:
Yes, it works already. Modern Windows accepts the '/' input as a directory separator in most places. We just had to cover the cases where '\' is used too.